scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -109,8 +115,9 @@
                                         "extensionSettings": {
                                             "dataCollectionSettings" : {
                                                 "interval": "[parameters('dataCollectionInterval')]",
-                                                 "namespaceFilteringMode": "[parameters('namespaceFilteringModeForDataCollection')]",
-                                                 "namespaces": "[parameters('namespacesForDataCollection')]"
+                                                "namespaceFilteringMode": "[parameters('namespaceFilteringModeForDataCollection')]",
+                                                "namespaces": "[parameters('namespacesForDataCollection')]",
+                                                "containerLogV2": "[parameters('enableContainerLogV2')]"
```

Comment:
since its Boolean flag, probable it will make sense to rename this from "containerLogV2"  to "enableContainerLogV2"

Reply:
CHANGED